example/build_cmake_installed_kk_as_language/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
-# Kokkos minimally requires 3.16 right now,
+# Kokkos minimally requires 3.22 right now,
 # but your project can set it higher
-cmake_minimum_required(VERSION 3.19)
```

Comment:
Note that this one was not 3.19 but I chose to bump it to the minimum instead of coming up with some newer version.